model-mesh/base/params.env

```diff
@@ -3,6 +3,6 @@ monitoring-namespace=
 odh-mm-rest-proxy=quay.io/opendatahub/rest-proxy:v0.11.0-alpha
 odh-modelmesh-runtime-adapter=quay.io/opendatahub/modelmesh-runtime-adapter:v0.11.0-alpha
 odh-modelmesh=quay.io/opendatahub/modelmesh:v0.11.0-alpha
-odh-openvino=quay.io/opendatahub/openvino_model_server:2022.3-gpu
+odh-openvino=quay.io/opendatahub/openvino_model_server@sha256:20dbfbaf53d1afbd47c612d953984238cb0e207972ed544a5ea662c2404f276d
```

Comment:
why is there sometime tag-notation and sometimes @sha256 digest notation in the manifests? I don't see a clear line of thought here.

Reply:
params.env file is for optional input from kfdef file. Openvino is not changed frequently so it is ok to use sha256 directly. however, for the others, we are thinking to use the doc to support a disconnected environment. In other word,  the image information in this file is just for default value but we can change it with sha256 with kfdef to support a disconnected environment.